### PR TITLE
Pdf exporter - delete xmp actions in the menu bar and the cli option

### PR DESCRIPTION
**Pull request by johannes-manner**
The idea is, to use a new exporter for writting bib entries to pdf metadata.
It enables users to select entries and export these entries via File->Export Selected Entries to a pdf file.
This exporter can also be used within the -o cli infrastructure.

java -jar JabRef.jar -o   path/article.pdf,pdf   path/entries.bib

Therefore, the xmp cli and the xmp actions in the menu bar can be deleted.

Opinions?

----

- [ ] Change in CHANGELOG.md described
- [ ] Tests created for changes
- [ ] Manually tested changed features in running JabRef
- [ ] Screenshots added in PR description (for bigger UI changes)
- [ ] Ensured that [the git commit message is a good one](https://github.com/joelparkerhenderson/git_commit_message)
- [ ] Check documentation status (Issue created for outdated help page at [help.jabref.org](https://github.com/JabRef/help.jabref.org/issues)?)

### CODE REVIEW

**Comment by tobiasdiez** [comment]
I think having a `PdfExporter` (maybe rename to `XmpPdfExporter`?) is a good idea, especially for the cli support we get in this way. However, I would keep the "Write XMP" action in the menu. Personally, I would never look under "Export" when I want to to add metadata to my pdfs, although you are of course right that technically it is an export of some kind.

**Comment by johannes-manner** [comment]
I deleted the cli option -xmp (integrated with https://github.com/JabRef/jabref/pull/3756), because the new XmpPdfExporter now enables the cli interaction via the -o exporter mechanism.

File: src/main/java/org/jabref/logic/exporter/XmpPdfExporter.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    private final XmpPreferences xmpPreferences;
+
+    public XmpPdfExporter(XmpPreferences xmpPreferences) {
+        super("pdf", FileType.PDF_XMP.getDescription(), FileType.PDF_XMP);
+        this.xmpPreferences = xmpPreferences;
+    }
+
+    @Override
+    public void export(BibDatabaseContext databaseContext, Path pdfFile, Charset encoding, List<BibEntry> entries) throws Exception {
+        if (pdfFile.toString().endsWith(".pdf")) {
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/logic/exporter/XmpPdfExporter.java]: Is this necessary? I think for an export it should not play a role if my flie is named  `test.asdf` for export, as long as the format is correct?

**Comment by Siedlerchr** [comment]
Please have a look at the faling exporter test. Then we can merge

**Pull request merged**